src/components/styles/bookCard.css

```diff
@@ -1,8 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,300;1,700&family=Montserrat:wght@700&family=Poppins:ital,wght@1,200;1,700&display=swap');
+
 .card {
   display: flex;
+  background-color: #fff;
+  align-items: center;
+  padding: 28px 40px 38px 0;
+  border-radius: 3px;
+  font-size: 17px;
   justify-content: space-around;
 }
 
-.progress {
+.category {
+  font-family: 'Montserrat', sans-serif;
+  opacity: 0.5;
+  font-size: 16px;
+}
+
+.title {
+  font-size: 26px;
+}
+
+.update-progress {
+  width: 11.5rem;
+  height: 2.063rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  color: white;
+}
+
+.percentage {
+  font-size: 2rem;
+}
+
+.chapter-details > p {
+  font-size: 19px;
+}
+
+.count > span,
+.current-chapter {
+  color: grey;
+  font-size: 18px;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+.actions > button {
+  border: none;
+  background: none;
+}
+
+.actions > button,
+.author {
+  color: #7bc0f5;
+  font-size: 17px;
+}
+
+.book-details {
+  display: flex;
+  flex-direction: column;
+  gap: 6px;
+}
+
+.details {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
```

Comment:
- [ ] Let's improve the styling of this section. The font-family for the category should be `Montserrat`. For the book title and the three buttons below the title, it should be `RobotoSlab` according to the design, please refactor this. Screenshots below will help show the differences in the font-family:

  | Design | You Implementation |
  | --- | --- |
  | ![image](https://user-images.githubusercontent.com/45002654/155511647-2caa1db7-f776-4c17-8d09-8365d9342749.png) | ![image](https://user-images.githubusercontent.com/45002654/155511687-f449394a-0868-42b2-b2ed-f02b08e7ecf9.png) |